### PR TITLE
Fix conversions against travis

### LINKED ISSUE #7083: Travis conversion generation verification failing on master for go 1.5
seeing diffs:
```
--- /home/travis/gopath/src/github.com/openshift/origin/pkg/api/v1/conversion_generated.go	2016-02-06 04:20:45.000000000 +0000
+++ /home/travis/gopath/src/github.com/openshift/origin/_output/verify-generated-conversions/pkg/api/v1/conversion_generated.go	2016-02-06 04:25:29.924682027 +0000
@@ -6759,7 +6759,7 @@ func autoConvert_api_PodTemplateSpec_To_
 	if err := Convert_api_ObjectMeta_To_v1_ObjectMeta(&in.ObjectMeta, &out.ObjectMeta, s); err != nil {
 		return err
 	}
-	if err := apiv1.Convert_api_PodSpec_To_v1_PodSpec(&in.Spec, &out.Spec, s); err != nil {
+	if err := Convert_api_PodSpec_To_v1_PodSpec(&in.Spec, &out.Spec, s); err != nil {
 		return err
 	}
 	return nil
```

### PR DESCRIPTION
**Pull request by smarterclayton**
The second commit is the easiest short term fix (makes order stable on both go versions) but I'm not sure it's the long term solution.

Fixes #7083

### CODE REVIEW

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
@liggitt or @deads2k

**Comment by smarterclayton** [comment]
[merge]

File: Godeps/_workspace/src/k8s.io/kubernetes/pkg/apis/extensions/install/install.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/golang/glog"
 
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/install"
```
Comment (deads2k) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/apis/extensions/install/install.go]: I'd expect this to get some serious push back.  Which parts are they dependent upon?

Comment (smarterclayton) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/apis/extensions/install/install.go]: Basically, the design of "install" being order independent due to packaging order means that unless you take this dependency you can't order installs.  So you have to have some class that registers them all in order, or add this dependency.  This is why we don't do static initializers.

**Pull request merged**